pkg/types/trade.go

```diff
@@ -95,6 +95,8 @@ type Trade struct {
 
 	// PnL is the profit and loss value of the executed trade
 	PnL sql.NullFloat64 `json:"pnl" db:"pnl"`
+
+	InsertedAt Time `db:"inserted_at"`
```

Comment:
also add a json tag here?

Reply:
@c9s It's not returned by the server json. The value is generated by mysql. Do we still have to add json tag?

Reply:
yeah we have some api renders the trade object in JSON format, but it's optional, not a problem